.github/workflows/deploy-local.yml

```diff
@@ -0,0 +1,51 @@
+name: Run Local Build
+
+on:
+  push:
+  workflow_dispatch: {}
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+  run-local-build:
+    needs: prepare
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install lcov
+      run: |
+        sudo apt-get install lcov
+      id: lcov
+    - name: Install Foundry
+      uses: foundry-rs/foundry-toolchain@v1
+      with:
+        version: nightly
```

Comment:
do we want to pin a version?

Reply:
pinned 1.2.0

Reply:
ahh, this doesn't work

only nightly is supported

per their docs;
`Version to install, e.g. nightly or 1.0.0. Note: Foundry only has nightly builds for the time being.`

All other versions 404

Reply:
Hmm iirc they keep monthly versions around. But latest is fine actually it’s actually good to make sure our stuff is deployable with latest foundry